modules/comparison-of-solutions-to-provision-node-local-storage.adoc

```diff
@@ -0,0 +1,172 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent_storage_local/overview-and-comparison-of-local-storage-options
+
+:_mod-docs-content-type: CONCEPT
+[id="comparison-of-solutions-to-provision-node-local-storage_{context}"]
+= Comparison of LVM Storage, LSO, and HPP
```

Comment:
This is not the biggest deal but I think it looks a little awkward to have this heading followed by another heading with no text in between. If you can, could you add some content here? Here's one simple sentence:
```suggestion
= Comparison of LVM Storage, LSO, and HPP

The following sections compare different aspects of the supported storage types.
```

Reply:
Added the following info:

"The following sections compare the functionalities provided by {lvms}, Local Storage Operator (LSO), and HostPath Provisioner (HPP) to provision local storage."